filebeat/docs/modules/cisco.asciidoc

```diff
@@ -12,8 +12,14 @@ This file is generated! See scripts/docs_collector.py
 
 == Cisco module
 
+//temporarily override modulename to create working link
+:modulename: cisco_asa
+
 include::{libbeat-dir}/shared/integration-link.asciidoc[]
 
+//reset modulename
+:modulename: cisco
+
```

Comment:
This seems like a reasonable work around for cases where there is one module but the integrations have been split into distinct packages since it takes people to the first in the family and the tree view is open so they can find the other members of the family of packages.

In the modules that are lexically later, this is less obvious since the opened tree view is not visible on the page; the Threatintel module is an example of this where the link takes the user to the AbuseCH page and the ThreatIntel integrations are off the page. This could be confusing for some users. I'm not sure what can be done about this. (Maybe  file an issue to jump the side panel to the integration that has been opened?)